From 8fbe6eb026ed37a8154dea15d55ab728a01f6f36 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:29:39 -0400
Subject: [alsa] Kconfig: rename HAS_IOPORT to HAS_IOPORT_MAP

Message-id: <140972823573-257-patchset-jkysela@redhat.com>
Patchwork-id: 90957
O-Subject: [RHEL 7.1 PATCH 256/442] ALSA: Kconfig: rename HAS_IOPORT to HAS_IOPORT_MAP
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

If the renamed symbol is defined lib/iomap.c implements ioport_map and
ioport_unmap and currently (nearly) all platforms define the port
accessor functions outb/inb and friend unconditionally.  So
HAS_IOPORT_MAP is the better name for this.

Consequently NO_IOPORT is renamed to NO_IOPORT_MAP.

The motivation for this change is to reintroduce a symbol HAS_IOPORT
that signals if outb/int et al are available.  I will address that at
least one merge window later though to keep surprises to a minimum and
catch new introductions of (HAS|NO)_IOPORT.

The changes in this commit were done using:

$ git grep -l -E '(NO|HAS)_IOPORT' | xargs perl -p -i -e 's/\b((?:CONFIG_)?(?:NO|HAS)_IOPORT)\b/$1_MAP/'

Signed-off-by: Uwe Kleine-Koenig <u.kleine-koenig@pengutronix.de>
Acked-by: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

(picked from kernel: ce816fa88cca083c47ab9000b2138a83043a78be)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/isa/Kconfig b/sound/isa/Kconfig
index affa134..0216475 100644
--- a/sound/isa/Kconfig
+++ b/sound/isa/Kconfig
@@ -191,7 +191,7 @@ config SND_ES18XX
 
 config SND_SC6000
  tristate "Gallant SC-6000/6600/7000 and Audio Excel DSP 16"
- depends on HAS_IOPORT
+ depends on HAS_IOPORT_MAP
  select SND_WSS_LIB
  select SND_OPL3_LIB
  select SND_MPU401_UART
diff --git a/sound/pci/Kconfig b/sound/pci/Kconfig
index 98a4591..3bd0bb1 100644
--- a/sound/pci/Kconfig
+++ b/sound/pci/Kconfig
@@ -685,7 +685,7 @@ config SND_LOLA
 
 config SND_LX6464ES
  tristate "Digigram LX6464ES"
- depends on HAS_IOPORT
+ depends on HAS_IOPORT_MAP
  select SND_PCM
  help
    Say Y here to include support for Digigram LX6464ES boards.
-- 
1.7.1